k8s/modules-ingress-mainnet.tf

```diff
@@ -1,3 +1,28 @@
+############CID Checker#######################################
+module "ingress-kong_cid-checker-calibrationnet" {
```

Comment:
you have the same module name in the k8s/modules-ingress-dev.tf file.
you have to rename it.